src/LatestReviewList/Review.tsx

```diff
@@ -0,0 +1,123 @@
+import type { ReviewQuery as ReviewQueryType } from './__generated__/ReviewQuery.graphql';
+import type { ErrorBoundary } from 'react-error-boundary';
+import type { PreloadedQuery } from 'react-relay';
+
+import ReviewQuery from './__generated__/ReviewQuery.graphql';
+
+import React, { useEffect, useRef } from 'react';
+import { Badge, Box } from '@chakra-ui/layout';
+
+import { usePreloadedQuery, useQueryLoader } from 'react-relay';
+import { graphql } from 'babel-plugin-relay/macro';
+
+import Rating from 'Rating';
+import Suspense from '../Suspense';
+
+type LoadedProps = {
+    data: PreloadedQuery<ReviewQueryType>
+}
+
+function LoadedReview(props: LoadedProps) {
+    const data = usePreloadedQuery(
+        graphql`
+            query ReviewQuery {
+                 latestReviews{
+                     id
+                     rating
+                     content
+                     doctor{
+                         firstname
+                         lastname
+                     }
+                     patient{
+                         firstname
+                         lastname
+                     }
+                 }
+            }
+        `,
+        props.data,
+    );
+
+    return (<>
```

Comment:
Q1: Is there a fixed number of reviews returned in every request? (if yes everything is fine)
Q2: What happens when you resize the screen? Is everything still ok?

Reply:
Q1-> at the moment we provide just 5 reviews. It will be updated and it will return the latest review based on the createdAt field.
Q2-> it seems okay. But yeah it is open to be designed and optimized, just i did not want to put too much effort on designing.

## Figure when resizing the screen
![debug](https://user-images.githubusercontent.com/26312757/121445864-978de880-c992-11eb-89c7-de9e08e6221e.png)

Reply:
Ok, that's great then